packages/mco/components/modals/app-manage-policies/helper/policy-config-viewer.tsx

```diff
@@ -0,0 +1,205 @@
+import { useCustomTranslation } from '@odf/shared/useCustomTranslationHook';
+import { StatusIconAndText } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  DescriptionList,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  SelectOption,
+  Text,
+} from '@patternfly/react-core';
+import '../style.scss';
+
+const dropdownOptions = (
+  placementControlInfo: PlacementControlProps,
+  defaultOptionText: string
+): JSX.Element[] => [
+  <SelectOption key={defaultOptionText} value={defaultOptionText} />,
+  ...Object.keys(placementControlInfo).map((placement) => (
+    <SelectOption key={placement} value={placement} />
+  )),
+];
+
+const getLabels = (
+  placementControlInfo: PlacementControlProps,
+  selected: string,
+  isDefaultSelected: boolean
+) =>
+  isDefaultSelected
+    ? // Aggregate all labels
+      Object.values(placementControlInfo || {}).reduce(
+        (acc, info) => [...acc, ...info?.pvcSelectors],
+        []
+      )
+    : // Get all labels from selected placement control info
+      [...placementControlInfo?.[selected]?.pvcSelectors];
+
+export const DescriptionListItem: React.FC<DescriptionListItemProps> = ({
+  term,
+  description,
+}) => {
+  return (
+    <DescriptionListGroup>
+      <DescriptionListTerm className="text-muted">{term}</DescriptionListTerm>
+      <DescriptionListDescription>{description}</DescriptionListDescription>
+    </DescriptionListGroup>
+  );
+};
+
+export const PolicyConfigViewer: React.FC<PolicyConfigViewerProps> = ({
+  policyName,
+  replicationType,
+  syncInterval,
+  isPolicyValidated,
+  clusters,
+  placementControlInfo,
+  defaultSelectionText,
+  isDisabledSelector,
+  onSelect,
+}) => {
+  const { t } = useCustomTranslation();
+  const [selected, setSelected] = React.useState(defaultSelectionText || '');
+  const isDefaultSelected = selected === defaultSelectionText;
+  const labels = React.useMemo(
+    () => getLabels(placementControlInfo, selected, isDefaultSelected),
+    [selected, isDefaultSelected, placementControlInfo]
+  );
+  const [unit, interval] = parseSyncInterval(syncInterval);
+  const syncScheduleFormat = {
+    m: t('minutes'),
+    h: t('hours'),
+    d: t('days'),
+  };
+
+  return (
+    <div className="mco-manage-policies__row--padding">
+      <div className="mco-manage-policies__header">
+        <Text component="h3"> {t('Policy configuration details')} </Text>
+        {!!Object.keys(placementControlInfo)?.length && (
+          <SingleSelectDropdown
+            id="placement-control-view-dropdown"
+            selectedKey={selected}
+            isDisabled={isDisabledSelector}
+            selectOptions={dropdownOptions(
+              placementControlInfo,
+              defaultSelectionText
+            )}
+            onChange={(value: string) => {
+              setSelected(value);
+              onSelect(value);
+            }}
+            className="mco-manage-policies__dropdown--width"
```

Comment:
```suggestion
            className="mco-manage-policies__dropdown--wide"
```
Modifier should be an adjective.